src/style.css

```diff
@@ -1,3 +1,78 @@
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  background-color: #fff;
+  color: #00acdc;
+}
+
+header {
+  min-width: 1024px;
+  max-width: 1200px;
+  flex-shrink: 0;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  justify-content: space-around;
+  align-items: center;
+  align-content: space-around;
+  flex-direction: row;
+  margin: 0 auto;
+  padding: 0 50px;
+  display: flex;
+}
+
+.moyo_logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+  margin: 10px 50px;
+}
+
+.nav {
+  display: inline-block;
+  justify-content: flex-end;
+}
```

Comment:
I think it doesn't work

![image](https://github.com/mate-academy/layout_moyo-header/assets/84296041/ba3aeaee-2d2a-4581-b053-68c29c24e746)